addons/l10n_in_withholding/wizard/l10n_in_withhold_wizard.xml

```diff
@@ -20,11 +20,6 @@
                     <field name="warning_message"/>
                 </div>
                 <sheet>
-                    <field name="related_move_id" invisible="1"/>
```

Comment:
@hamo-odoo It would be good to check that the wizard is still working with this.